sync/p2p_messages.md

```diff
@@ -139,6 +150,36 @@ Message is RLP encoded, fields:
 A signed transaction is serialized as a tagged and versioned
 [signed transaction](../serializations.md#signed-transaction).
 
+## MSG_BLOCK_TXS
+*(Tag = 13)
+
+Message is RLP encoded, fields:
```

Comment:
This is correct:
https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_peer_connection.erl#L1004-L1005